### PR TITLE
Repo maintenance

### PR DESCRIPTION
**Pull request by jenniferboedker**
Improve readme and githubactions

### CODE REVIEW

File: .github/CODEOWNER [hunk 1]
```diff
@@ -0,0 +1 @@
+* @qbicsoftware/itss
```
Comment (jenniferboedker) [inline .github/CODEOWNER:1]:
```suggestion
* @qbicsoftware/itss

```

File: README.md [hunk 2]
```diff
@@ -5,17 +5,17 @@ that have received a new status. This is only done for the current day the tool
 People who are subscribed are notified about their projects' changes
 will be notified via email. This service creates the messages and sends them.
 
-## Requirements
+[![Build Maven Package](https://github.com/qbicsoftware/sample-notificator-cli/actions/workflows/build_package.yml/badge.svg)](https://github.com/qbicsoftware/spring-boot-rest-service-template/actions/workflows/build_package.yml)
```
Comment (Steffengreiner) [inline README.md:8]:
```suggestion
[![Build Maven Package](https://github.com/qbicsoftware/sample-notificator-cli/actions/workflows/build_package.yml/badge.svg)](https://github.com/qbicsoftware/sample-notificator-cli/actions/workflows/build_package.yml)
```

File: README.md [hunk 3]
```diff
@@ -5,17 +5,17 @@ that have received a new status. This is only done for the current day the tool
...
+[![Run Maven Tests](https://github.com/qbicsoftware/sample-notificator-cli/actions/workflows/run_tests.yml/badge.svg)](https://github.com/qbicsoftware/spring-boot-rest-service-template/actions/workflows/run_tests.yml)
```
Comment (Steffengreiner) [inline README.md:9]:
```suggestion
[![Run Maven Tests](https://github.com/qbicsoftware/sample-notificator-cli/actions/workflows/run_tests.yml/badge.svg)](https://github.com/qbicsoftware/sample-notificator-cli/actions/workflows/run_tests.yml)
```

File: README.md [hunk 4]
```diff
@@ -5,17 +5,17 @@ that have received a new status. This is only done for the current day the tool
...
+[![CodeQL](https://github.com/qbicsoftware/sample-notificator-cli/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/qbicsoftware/spring-boot-rest-service-template/actions/workflows/codeql-analysis.yml)
```
Comment (Steffengreiner) [inline README.md:10]:
```suggestion
[![CodeQL](https://github.com/qbicsoftware/sample-notificator-cli/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/qbicsoftware/sample-notificator-cli/actions/workflows/codeql-analysis.yml)
```

File: README.md [hunk 5]
```diff
@@ -5,17 +5,17 @@ that have received a new status. This is only done for the current day the tool
...
+[![release](https://img.shields.io/github/v/release/qbicsoftware/sample-notificator-cli?include_prereleases)](https://github.com/qbicsoftware/spring-boot-rest-service-template/releases)
```
Comment (Steffengreiner) [inline README.md:11]:
```suggestion
[![release](https://img.shields.io/github/v/release/qbicsoftware/sample-notificator-cli?include_prereleases)](https://github.com/qbicsoftware/sample-notificator-cli/releases)
```

File: README.md [hunk 6]
```diff
@@ -5,17 +5,17 @@ that have received a new status. This is only done for the current day the tool
...
 
-### Database
+[![license](https://img.shields.io/github/license/qbicsoftware/sample-notificator-cli)](https://github.com/qbicsoftware/spring-boot-rest-service-template/blob/main/LICENSE)
```
Comment (Steffengreiner) [inline README.md:13]:
```suggestion
[![license](https://img.shields.io/github/license/qbicsoftware/sample-notificator-cli)](https://github.com/qbicsoftware/sample-notificator-cli/blob/main/LICENSE)
```

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Nicely done! However there seems to be a copy paste issue with the badge links. Otherwise it looks good!

File: README.md (hunk 2 above)
Comment (KochTobi) [reply README.md:8]: you can `command` + `R` this easily 😄

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Well done! 
Once the PR workflow is gone i'll approve it 👍

**Review by Steffengreiner — APPROVED** [review]
Perfect

**Pull request merged**